app/uk/gov/hmrc/cardpaymentfrontend/models/extendedorigins/ExtendedOrigin.scala

```diff
@@ -133,3 +134,87 @@ trait ExtendedOrigin {
   def surveyBannerTitle: String
 
 }
+
```

Comment:
This is to replace OriginExtraInfo that was being injected everywhere, it should always have been an extension on origin, we just never found the time.

Reply:
Plus it lets us just do origin.lift instead of having to inject etc